docs/monthly-meeting/2022-12.md

```diff
@@ -0,0 +1,84 @@
+
+* (Hugo) [Add release cycle chart to docs](https://github.com/python/docs-community/issues/67):
+  * [CSV PR](https://github.com/python/devguide/pull/884) was merged
+  * [charts PR](https://github.com/python/devguide/pull/988) ready for review -> merged during the meeting :rocket:
+  * [demo](https://cpython-devguide--988.org.readthedocs.build/versions/#python-release-cycle)
+  * Followup: @CAM-Gerlach opened devguide issues for adding the additional development phase transition dates [python/devguide#998](https://github.com/python/devguide/issues/998) & a table of individual minor versions [python/devguide#999](https://github.com/python/devguide/issues/999)
+  * Followup: Hugo will check with it no JavaScript, and the remote hosting of https://unpkg.com/mermaid/dist/mermaid.min.js in case it goes down
+    * -> No JS PR: https://github.com/python/devguide/pull/997
+
+* https://packaging.python.org/en/latest/glossary/
+  * @CAM-Gerlach to improve the glossary. It can then be linked or copied to CPython docs.
+
+### PEP Workflow
+
+* [PEP 1 rewrite?](https://discuss.python.org/t/21068/26)
+  * It was generally agreed that PEP 1 (and PEP 12) could use significant re-organization and trimming.
+  * Petr suggests moving the author-facing help out of PEP 1/to the devguide
+  * CAM mentions [python/peps#2337](https://github.com/python/peps/issues/2337) which proposes referencing the Sphinx reST primer instead of duplicating information there in PEP 12 and consolidating normative process guidance in PEP 1 and author-facing info in PEP 12/the devguide
+  * @CAM-Gerlach will open a PR implementing that first step, linking the Sphinx reST primer in PEP 12 instead of duplicating that information
+  * CAM also mentions the devguide's reST primer is essentially just a less-up-to-date copy of the Sphinx one; it should be revised to link to it and focus on CPython-specific guidance. This issue was already open as [python/docs-community#57](https://github.com/python/docs-community/issues/57), which was moved to [python/devguide#1000](https://github.com/python/devguide/issues/1000), where @CAM-Gerlach will work on it
+* Final PEPs and the canonical docs banner link
+  * [python/peps#2719](https://github.com/python/peps/issues/2719) will improve the existing banners directing users to the canonical docs for Final PEPs added in [python/peps#2702](https://github.com/python/peps/pull/2702) to be sticky at the top; @CAM-Gerlach to submit PR
+  * We can start going through Final PEPs and systemically marking them with the new banner as appropriate
+  * @CAM-Gerlach will submit PRs for [python/peps#2872](https://github.com/python/peps/issues/2872) to mark old implemented Accepted PEPs as Final, and will add banner in the process
+  * We might need a checklist for marking PEPs Final
+    * e.g.: is it documented outside the PEP? Add a note?
+    * @CAM-Gerlach will open an issue to discuss adding checklists for new Approved/Rejected and Final PEPs
```

Comment:
```suggestion
    * @CAM-Gerlach opened issue [python/peps#2937](https://github.com/python/peps/issues/2937) to discuss adding checklists for new Approved/Rejected and Final PEPs
```

Reply:
Generally, meeting minutes should record what happened during the meeting, so putting this action point in past tense makes it sound like it had already been done before the meeting.

Maybe keep it future tense, and add a little followup note to say it's now been done?

Reply:
Right, but I saw that you added a bunch of things after so... :stuck_out_tongue_winking_eye: 

In any case, I removed it inline and added it in a paren.